### ISSUE TITLE
Show error message on running your app

### ISSUE THREAD

**Issue by joumduk** [inspireui/support#1589]
```
Compiler message:
../../../development/flutter/.pub-cache/hosted/pub.dartlang.org/cached_network_image-1.1.3/lib/src/cached_network_image_provider.dart:42:24: Error: The method 'CachedNetworkImageProvider.load' has fewer positional arguments than those of overridden method 'ImageProvider.load'.
  ImageStreamCompleter load(CachedNetworkImageProvider key) {
                       ^
../../../development/flutter/packages/flutter/lib/src/painting/image_provider.dart:403:24: Context: This is the overridden method ('load').
  ImageStreamCompleter load(T key, DecoderCallback decode);
                       ^
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
Failed to build bundle.
Error launching application on iPhone 11 Pro Max.

```

When I download and run, it shows those messages  on debug console
how to solve this problem??

**Comment by minhcasi** [comment]
@joumduk - Try to run "flutter clean" and "flutter channel stable" then reinstall the product.

Could you send us the result from "flutter doctor -v"

**Comment by minhcasi** [comment]
Also, Could you help to share the screenshot of the purchase screenshot from http://codecayon.net/downloads  it's used to verify your proof of purchase? sorry for any inconvenience - Thank you!

**Comment by joumduk** [comment]
I download again and install.
Then,  run "flutter clean" and "flutter channel stable" , but still show same error message 
'flutter doctor -v' show like this

`$ flutter doctor -v
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.15.1 19B88, locale ko-TH)
    • Flutter version 1.12.13+hotfix.5 at /Users/peter/development/flutter
    • Framework revision 27321ebbad (2 days ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/peter/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.2.1, Build version 11B500
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.41.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[✓] Connected device (1 available)
    • iPhone 11 Pro Max • 1288793F-5E90-4BD5-BF93-62D0C1B639CE • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-2 (simulator)

• No issues found!


$ flutter channel stable
Switching to flutter channel 'stable'...
git: Already on 'stable'
git: Your branch is up to date with 'origin/stable'.
`

**Comment by minhcasi** [comment]
Please try to update the app to the latest version 1.6, there is some bug fix and compatible with Flutter 1.12 - Detail changelog will be updated from https://medium.com/@inspireui/announcing-fluxstore-1-6-337efff1791f - thank you!
